client/coral-admin/src/routes/Moderation/containers/Moderation.js

```diff
@@ -212,7 +212,7 @@ const withModQueueQuery = withQuery(gql`
     }) {
       ...CoralAdmin_Moderation_CommentConnection
     }
-    assets: assets(query: {}) {
+    assets: assets(query: {limit: 10}) {
```

Comment:
Does it? Even when all arguments are defined optional? There must be some easier way to specify it :thinking: than `assets: assets(query: null)`.